data/jobs.txt

```diff
@@ -1477,6 +1477,76 @@ mission "Core Pirate Attack"
 	on complete
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
+		
+mission "Raider Attack 1"
+	name "Defend <planet>"
+	description "Defeat an alien raid on <destination>."
+	minor
+	repeat
+	to offer
+		"combat rating" > 550
+		random < 7
+		has "main plot completed"
+	source
+		planet "Sundive"
+		planet "Shangri-La"
+		planet "Nimbus"
+	npc evade
+		government "Korath"
+		personality target timid unconstrained uninterested harvests plunders
+		fleet "Korath Raid" 2
+	npc
+		government "Syndicate"
+		personality heroic staying vindictive
+		fleet "Large Syndicate"
+		fleet "Small Syndicate"
+	on offer
+		conversation
+			`Suddenly you hear raised voices and shouting outside: "We are under attack! <planet> is under attack by alien raiders! We need every combat-worthy ship to join in the defenses!" The authorities will probably pay you quite well if you assist them, but this could also be an easy way to get yourself killed.`
```

Comment:
"Defenses" should be "defenses" here.

Reply:
It would be, except that all the other planetary defense missions use "defenses".

Reply:
That's why I recommended changing the rest in https://github.com/endless-sky/endless-sky/pull/3468.